lab-will/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "lab-will",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "start": "clear\nDEBUG='user*' node server.js",
```

Comment:
These lines in here have to match the name you are using for the debug in the files where you are including it.  Here you are setting it to look for a user* but in your files you are calling everything cfgram.